dandi/files/bids.py

```diff
@@ -87,7 +86,7 @@ def _validate(self) -> None:
                         bids_paths += [str(readme_candidate)]
                 # end of ad-hoc fix.
 
-                results = validate_bids(self.bids_root)
+                results = bids_validate(self.bids_root)
```

Comment:
so here, if input path was not the top of the bids dataset -- issue a lgr.WARNING that we will use bidsschematools validation only per each path, and it would validate only the path names. To validate full bids dataset -- point to the top of the bids dataset.